hoodie-utilities/pom.xml

```diff
@@ -283,6 +283,13 @@
             <artifactId>bijection-avro_2.10</artifactId>
             <version>0.9.2</version>
         </dependency>
+        <dependency>
```

Comment:
Please file a follow ticket to remove this and switch to actual maven artifact later